### PR TITLE
939 - Update deploy script and infrastructure readme

### PR DESCRIPTION
**Pull request by avrohomgottlieb**
## Issue Number

Closes https://github.com/AlexsLemonade/scpca-portal/issues/939

## Purpose/Implementation Notes

This PR makes slight corrections to the deploy script and infrastructure readme in order to enable the spinning up of a dev stack.

## Types of changes

What types of changes does your code introduce?

- Bugfix (non-breaking change which fixes an issue)
- Refactor (addresses code organization and design mentioned in corresponding issue)

## Functional tests

N/A

## Checklist

- [x] Lint and unit tests pass locally with my changes

## Screenshots

N/A

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -128,6 +128,13 @@ infrastructure/.terraform.lock.hcl
 .vscode
 *.code-workspace
 
+# SSH keys
+*.pem
+*.pub
+
+# 1Password integration
+.op/
+
```
Comment (davidsmejia) [inline .gitignore:137]: Can you tell me what step created this folder?

Comment (avrohomgottlieb) [reply .gitignore:137]: When we configured the `op` plugin, it created a `.op/` directory, which contains a `plugins/` directory, which in turn includes a `ccdl.json` file. This json file includes my op `account_id`, the plugin `entrypoint`, and a field called `credentials` which is an object that includes the type of plugin, the credential_type (which is an access_key), my vault_id, the item_id, etc.

Comment (davidsmejia) [reply .gitignore:137]: Ok so I think that makes sense to keep we might want to add this to documentation for a .gitignore defaults file.

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**